fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/filter/AlphanumericLengthFilter.java

```diff
@@ -55,37 +54,38 @@ public final class AlphanumericLengthFilter extends FilteringTokenFilter {
 
     @Override
     protected boolean accept() {
-        String typeAttStr = typeAtt.type();
-        if (UAX29URLEmailTokenizer.TOKEN_TYPES[UAX29URLEmailTokenizer.IDEOGRAPHIC].equals(typeAttStr) ||
-                UAX29URLEmailTokenizer.TOKEN_TYPES[UAX29URLEmailTokenizer.HANGUL].equals(typeAttStr) ||
-                UAX29URLEmailTokenizer.TOKEN_TYPES[UAX29URLEmailTokenizer.HIRAGANA].equals(typeAttStr) ||
-                UAX29URLEmailTokenizer.TOKEN_TYPES[UAX29URLEmailTokenizer.KATAKANA].equals(typeAttStr)
-        ) {
-            return true;
-        } else {
-            int len = termAtt.length();
-            return (len >= minAlphanumLength && len <= maxAlphanumLength);
-        }
+        return CjkUnigramFilter.isUnigramTokenType(typeAtt.type())
```

Comment:
Should this effectively have the same behavior if you are using `CjkUnigramFilter` around `UAX29URLEmailTokenizer`? Or is there some backwards compatibility issues in the tokens produced?

Reply:
I guess this length filter was written to specifically work with the CjkUnigramFilter. It would not be able to distinguish unigrams produced by other tokenizers